.github/workflows/test.yml

```diff
@@ -249,11 +257,14 @@ jobs:
       OVN_MULTICAST_ENABLE:  "false"
     steps:
     - name: Set up Go
-      uses: actions/setup-go@v1
+      uses: actions/setup-go@v2
       with:
         go-version: ${{ env.GO_VERSION }}
       id: go
 
+    - name: Check out code into the Go module directory
+      uses: actions/checkout@v2
+
```

Comment:
:raising_hand_man: I could not come up with a reason on why we need this checkout

Reply:
+1 I don't believe we do